pkg/env/env.go

```diff
@@ -143,18 +146,50 @@ func (v *ViperEnv) setupConfigFile() error {
 		}
 		return fmt.Errorf("could not read configuration cfg: %s", err)
 	}
+
+	v.Viper.WatchConfig()
+
+	dynamicLogHandler := func(env Environment) func(event fsnotify.Event) {
+		return func(event fsnotify.Event) {
+			if strings.Contains(event.String(), "WRITE") || strings.Contains(event.String(), "CREATE") {
+				logLevel := env.Get("log.level").(string)
+				logFormat := env.Get("log.format").(string)
+
+				log.C(ctx).Warnf("Reconfiguring logrus logging using level %s and format %s", logLevel, logFormat)
+				var err error
+				ctx, err = log.Configure(ctx, &log.Settings{
+					Level:  logLevel,
+					Format: logFormat,
+					Output: os.Stdout.Name(),
```

Comment:
if output change is forbidden at runtime this may override the default one

Reply:
true - a more general question would be:

Does it make sense to allow log output switching via the apploication.yml file ? If you have access to the file system you could switch the output to a file (if we add support for files as output) which might or might not be helpful?

Or we can just forbid changing the output from the application.yml as well

Reply:
Unless we add the possibility to log to a file I don't think we need to support changing the output